scripts/spec-check.py

```
@@ -0,0 +1,124 @@
+    if not text.startswith("---\n"):
+        fail(["缺 YAML frontmatter（必须以 --- 开头）"])
+    parts = text[4:].split("\n---", 1)
+    if len(parts) < 2:
+        fail(["frontmatter 未闭合（缺结束 ---）"])
+    try:
+        fm = yaml.safe_load(parts[0])
+    except yaml.YAMLError as e:
+        fail([f"frontmatter YAML 解析失败: {e}"])
+    errs = []
+
+    # 1. 必备键（nonGoals 允许空列表——"无非目标"是合法态；其余为空即拒绝）
```

Ruff (0.16.1)

[warning] Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)